### DISCUSSION TITLE
新聞分享:特斯拉正在開發人型機器人，身高比馬斯克矮 15 公分

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS3073-C/Python-Community-GS3073-C#69] [科技新知]
馬斯克好高，人形機器人好酷，想買一台。
內容:https://technews.tw/2021/08/20/tesla-humanoid-robot/

**Comment by Timmy110403540** [comment]
走在科技尖端的馬斯克 竟然還會害怕人工智慧(機器人)!

**Comment by ccathyy** [comment]
希望能順利製造出原型機!期待明年有製造成功的好消息~製造成功的好消息~

**Comment by 110409538** [comment]
感覺這個機器人會很帥,打造得不像人,而更像機器,會有一種很科技的感覺,很酷.

**Comment by bonny123865** [comment]
會不會真的能像電影當中出現的未來人一樣處理危機？！
有點期待做出來的功能會是如何，當然還有那個螢幕！
我現在才知道馬斯克身高那麼高應該是正常的吧！

**Comment by wh20502** [comment]
馬克斯也非常有趣，害怕人工智慧，現在卻要做一台出來嚇自己，還規定不能比他高。

**Comment by ghost** [comment]
原來沒有比他高是因為害怕機器人阿XD不過很期待他們會做出什麼樣的機器人

**Comment by Shiba010** [comment]
特斯拉做的機器人真令人期待！
